src/filter.ts

```diff
@@ -288,6 +288,13 @@ const COMMON_MESSAGE_KEYS = {
     caption_entities: ENTITY_KEYS,
     caption: {},
 
+    link_preview_options: {
+        is_disabled: {},
```

Comment:
I think it is reasonable to remove it.

```suggestion
```

If you want, we can ask the TG devs about it first.